styles.css

```diff
@@ -1 +1,70 @@
+html {
+    background-color: pink;
+    
+    font-family: Georgia, 'Times New Roman', Times, serif;
+}
 
+.container {
+    margin: 100px 100px 100px 100px;
+    background-color: white;
+    background: url(background.jpg) no-repeat;
+    background-size: 1280px 768px;
+    width: 1280px;
+    height: 768px;
+}
+
+.navbar {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    text-align: right;
+    justify-content: flex-end;
+    align-items: center;
+}
+
+
+.navbaritem {
+    margin: 50px 40px 0px 40px;
+    color: black;
+}
+
+#LOGIN {
```

Comment:
DO NOT use the uppercase in ID or class name

Reply:
Oh ok this is an important one.